storage/buckets/disable_soft_delete_policy.go

```diff
@@ -37,26 +39,17 @@ func disableSoftDeletePolicy(bucketName string) error {
 
 	bucket := client.Bucket(bucketName)
 
-	// Get the current bucket attributes.
-	attrs, err := bucket.Attrs(ctx)
-	if err != nil {
-		return fmt.Errorf("bucket.Attrs: %w", err)
-	}
-
-	// Setting retention duration to 0 disables soft delete.
-	attrs.SoftDeletePolicy = &storage.SoftDeletePolicy{
-		RetentionDuration: time.Duration(0),
-	}
-
-	// Update the bucket with zero retention duration.
+	// Update the bucket with zero retention duration to disable soft delete.
 	_, err = bucket.Update(ctx, storage.BucketAttrsToUpdate{
-		SoftDeletePolicy: attrs.SoftDeletePolicy,
+		SoftDeletePolicy: &storage.SoftDeletePolicy{
+			RetentionDuration: time.Duration(0),
```

Comment:
Nit: I think this can just be represented as an empty soft delete policy because time.Duration is an int64 type, so it defaults to zero.